book/src/template_syntax.md

```diff
@@ -587,3 +587,49 @@ You can optionally specify the name of the macro in `endmacro`:
 ```html
 {% macro heading(arg) %}<p>{{arg}}</p>{% endmacro heading %}
 ```
+
+You can also specify arguments by their name (as defined in the macro):
+
+```
+{% macro heading(arg, bold) %}
+
+<h1>{{arg}} <b>{{bold}}</b></h1>
+
+{% endmacro %}
+
+{% call heading(bold="something", arg = "title") %}
```

Comment:
I think the documentation should be consistent in whether it uses spacing around `=` even though the parser should cope with both.

Reply:
Updated! I also precised that we can have whitespace characters around `=`.